microsetta_interface/templates/update_address.jinja2

```diff
@@ -35,6 +36,21 @@
                     $("#state").append(option);
                 }
             }
+        } else if (selected_country == "Spain") {
+                for (var state of SPAIN_STATES)
+                {
+                    var option = $('<option></option>')
+                        .attr("value", state[0])
+                        .text(state[1]);
+                    $("#state").append(option);
+                }
+                $("#form_group_address_3, #form_group_address_type").show();
+                $("#phone").rules("remove");
+                $("#phone").rules( "add", {
```

Comment:
Phone validation is here but it looks like the phone field isn't actually on the form - can you adjust that?